ci/build-freebsd.sh

```diff
@@ -22,7 +23,8 @@ meson setup build \
     $NULL
 
 meson compile -C build
-meson test -C build
+# the ffmpeg tests fail due to being too old; skip for now
+meson test -C build --no-suite ffmpeg
```

Comment:
We know the ffmpeg version in meson/test. Instead of hardcoding skipped test here. If if incompatible ffmpeg version is compiled in test should return `77` and meson would mark it as `SKIP` or do not define this test based on version. 

This will automagicaly start testing ffmpeg when it is updated and not require manual intervention.

Reply:
Went with just requiring a minimum libavutil verison for those specific tests.